kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -1,12 +1,12 @@
 # Authors: Edouard Oyallon, Joakim Anden, Mathieu Andreux
 BACKEND_NAME = 'numpy'
 
-from ...backend.numpy_backend import NumpyBaseBackend
+from ...backend.numpy_backend import NumpyBackend, complex_check, real_check
```

Comment:
If checks are included in the class, we wouldn't need these extra imports here.

Reply:
True.